index.html

```diff
@@ -49,13 +49,13 @@
 <div class="topic">
   <h2>Digital</h2>
   <p>Technology is all about finding the right tool for the job, and creating the tool if you can't find the one you want. I've planned an entire digital strategy and consulted on others, created email formats and content, and learned to code. I understand the entire pipeline, from back end development to getting the wording just right.</p>
-  <p>I'm a bit of a Twitter expert, having spent over a year studying and alayzing the Twitter ecosystem. That theoretical experience allows me to come up with new approaches to the medium when working with clients, including advising organizations on the best way to get their message trending and how to best interact with the fans they already have online. I've also taught others how to create a social media presense for their own organizations. </p>
+  <p>I'm a bit of a Twitter expert, having spent over a year studying and analyzing the Twitter ecosystem. That theoretical experience allows me to come up with new approaches to the medium when working with clients, including advising organizations on the best way to get their message trending and how to best interact with the fans they already have online. I've also taught others how to create a social media presence for their own organizations. </p>
   </div>
   <div class="topic">
     <h2>Data</h2>
     <p>My experience in data analysis is based on my interest in the subject matter. I fine-tuned a social media analytics program in order to help a variety of organizations get their message out, including using natural language processing to find out what wording was popular and using metadata to analyze the habits of their followers.
  </p>
- <p>Most of my coding experience has been the result of working with data. In Python I've learned how to use Pandas, NumPy, ScikitLearn, and assorted other packages. I've worked to develop skills in Data Visualization, as well.</p>
+ <p>Most of my coding experience has been the result of working with data. In Python I've learned how to use Pandas, NumPy, Scikit-Learn, and assorted other packages. I've worked to develop skills in Data Visualization, as well.</p>
   </div>
     <div class="topic">
     <h2>Organizing</h2>

```

Comment:
I can't seem to hit the line, but the next paragraph: "working with others to improve the lives of others" reads funnily to me.